### PR TITLE
Fixed endless spinner when no catalog items available

### PR DESCRIPTION
**Pull request by dtaylor113**
Fixes endless spinner issue for Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1501785

### CODE REVIEW

**Comment by dtaylor113** [comment]
> Does the empty state message flash on load,

Nope.  It only gets displayed when there are no items.

File: src/components/services-view/services-view.controller.ts [hunk 1]
```diff
@@ -63,6 +65,11 @@ export class ServicesViewController implements angular.IController {
       appliedFilters: []
     };
 
+    this.ctrl.emptyConfig = {
+      title: 'No Items',
+      info: 'There are no catalog items.'
+    };
```
[the remark this answers is not in the document]
Comment (dtaylor113) [reply src/components/services-view/services-view.controller.ts]: Ok, changed this PR to just fix the endless spinner and filed https://github.com/openshift/origin-web-console/issues/2496 to handle adding the empty state into console.

**Comment by dtaylor113** [comment]
@spadgett looking for a review when you get a moment  -thanks

**Comment by jwforres** [comment]
@spadgett needs follow-up review

**Comment by spadgett** [comment]
@dtaylor113 Were you going to make the console changes too to add the empty state message?

**Pull request merged**